conf/iptables.conf.example

```diff
@@ -116,8 +116,11 @@
 -A input-highavailability-if --protocol udp --match udp --dport 5407 --jump ACCEPT
 #DRBD
 -A input-highavailability-if --protocol tcp --match tcp --dport 7788 --jump ACCEPT
-# Heartbeat
```

Comment:
Since there is still reference to Heartbeat in the doc i should prefer to keep it.